### PR TITLE
 added tests for importing a partially encrypted private key 

### LINKED ISSUE #869: Test import a partially encrypted private key
Need to add a test for this issue

_Originally posted by @DenBond7 in https://github.com/FlowCrypt/flowcrypt-android/issues/708#issuecomment-607291974_

### PR DESCRIPTION
**Pull request by DenBond7**
This PR added tests for importing a partially encrypted private key 

close #869

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities

### CODE REVIEW

File: FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/CheckKeysFragment.kt [hunk 1]
```diff
@@ -380,6 +396,8 @@ class CheckKeysFragment : BaseFragment<FragmentCheckKeysBinding>() {
   }
 
   companion object {
+    private const val REQUEST_CODE_SINGLE_PARTIALY_ENCRTYPRED_KEY = 1
```
Comment (sosnovsky) [inline FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/CheckKeysFragment.kt:399]: `REQUEST_CODE_SINGLE_PARTIALY_ENCRTYPRED_KEY` -> `REQUEST_CODE_SINGLE_PARTIALLY_ENCRYPTED_KEY`

**Review by sosnovsky — APPROVED** [review]
All good, just a small typo 👍

File: FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/CheckKeysFragment.kt (hunk 1 above)
Comment (DenBond7) [reply FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/CheckKeysFragment.kt:399]: Interesting... Android Studio didn't show a warning about that. But usually shows. Thank you, I will fix it!

**Review by sosnovsky — APPROVED** [review]
👍

**Pull request merged**